day_1/parse_tree.rb

```diff
@@ -0,0 +1,85 @@
+require_relative 'parse_tag'
+
+Tag = Struct.new(:type, :classes, :id, :name, :src, :title, :text, :children, :parent)
+# :attributes that holds all html elements
+# Tag class
+# methods on tag class to retrieve that attribute
+
+class ParseTree
+	attr_accessor :html_code, :root, :tag_type
+
+  def initialize(html_code)
+  	regex = /<.*?>/
```

Comment:
It might be better to give the regular expression a name that describes its purpose, like tag_regex.

Also, since this represents an unchanging component, it might help to make it a class level constant.